packages/@aws-cdk-testing/framework-integ/test/aws-lambda-nodejs/test/integ.dependencies.ts

```diff
@@ -37,6 +37,9 @@ class SdkV3TestStack extends Stack {
     this.lambdaFunction = new lambda.NodejsFunction(this, 'external-sdk-v3', {
       entry: path.join(__dirname, 'integ-handlers/dependencies-sdk-v3.ts'),
       runtime: Runtime.NODEJS_18_X,
+      bundling: {
```

Comment:
Can you remove this so that the integ test will test the new behavior?

Can you also add `diffAssets: true` to the `IntegTest`

Reply:
I think it may be because there is no lock file. I would try adding a new sub folder for this handler with a minimal `package.json` and `package-lock.json`.

Reply:
That's exactly what I did 🤔

Reply:
Ok I think for this test we have to have the bundling actually perform an npm install.

```ts
    this.lambdaFunction = new lambda.NodejsFunction(this, 'external-sdk-v3', {
      entry: path.join(__dirname, 'integ-handlers/v3/dependencies-sdk-v3.ts'),
      runtime: Runtime.NODEJS_18_X,
      depsLockFilePath: path.join(__dirname, 'integ-handlers/v3/package-lock.json'),
      bundling: {
        bundleLambdaProvidedAwsSdk: true,
        commandHooks: {
          beforeInstall() {
            return [];
          },
          afterBundling() {
            return [];
          },
          beforeBundling(inputDir, _outputDir) {
            return [`cd ${inputDir}`, 'npm install --ci'];
          },
        },
      },
    });
```

Reply:
Okay that seems to work ✅
So I want to create the new snapshots now but that leads me into a rabbit hole 😄 
When running `yarn integ --update-on-failed aws-lambda-nodejs/test/integ.dependencies.js` it says for the test with the bundled sdk:
![Screenshot 2023-06-09 at 16 14 51](https://github.com/aws/aws-cdk/assets/16321871/b746a8b3-d513-4ea6-a691-d14a550d63fd)

It is the `Response object is too long` error which relates to several open issues. Any idea how to do a workaround for this?